parm/config/gfs/config.cleanup

```diff
@@ -0,0 +1,15 @@
+#! /usr/bin/env bash
+
+########## config.cleanup ##########
+echo "BEGIN: config.cleanup"
+
+# Get task specific resources
+source "${EXPDIR}/config.resources" cleanup
+
+export DELETE_COM_IN_ARCHIVE_JOB="YES"   # NO=retain ROTDIR.  YES default in cleanup.sh
```

Comment:
Should this be `DELETE_COM="YES"` and be moved to `config.base`?

Reply:
Renamed, but kept it here.